source/val/validate_annotation.cpp

```diff
@@ -427,6 +444,9 @@ spv_result_t AnnotationPass(ValidationState_t& _, const Instruction* inst) {
     case SpvOpGroupMemberDecorate:
       if (auto error = ValidateGroupMemberDecorate(_, inst)) return error;
       break;
+    case SpvOpDecorateStringGOOGLE:
```

Comment:
This MR does not work. When we get to this switch, always "ValidateDecorate" is the next line. Could you help me understand why "SpvOpDecorateStringGOOGLE" is never the case?

Reply:
I'm not sure what you mean here. When I test this method it works as its supposed to.